styles/global.css

```diff
@@ -14,40 +13,59 @@ body {
   font-family: Arial, Helvetica, sans-serif;
   line-height: 1.6;
 }
+
 .headerNav {
   display: flex;
   justify-content: space-between;
   align-items: center;
-  position: absolute;
   width: 100%;
-  /* make sure header always overlay bg image */
-  z-index: 2;
+  background-color: black;
+  position: fixed;
+  height: 60px;
 }
 
 .logo a {
   text-decoration: none;
-  color: black;
+  color: white;
 }
 .logo h1 {
-  font-size: 24px;
+  font-size: 18px;
   margin-bottom: 0;
   margin-left: 20px;
+  justify-items: center;
 }
 .logo p {
   margin-top: 0;
-  font-size: 14px;
+  font-size: 12px;
   text-align: center;
-  margin-left: 20px;
+  margin-left: 16px;
 }
 .navbar {
   display: flex;
   justify-content: space-between;
+  justify-items: center;
   padding-right: 40px;
-  gap: 30px;
+  /* gap: 30px; */
 }
+
 .navbar a {
   text-decoration: none;
-  color: black;
+  color: white;
+}
+
+.mystory-header,
```

Comment:
Can you try to use google font Hind 600 for header and Open Sans 400 for paragraph